lib/block_view.go

```diff
@@ -3443,6 +3490,17 @@ func (bav *UtxoView) _connectTransaction(txn *MsgDeSoTxn, txHash *BlockHash,
 	case TxnTypeUnjailValidator:
 		totalInput, totalOutput, utxoOpsForTxn, err = bav._connectUnjailValidator(txn, txHash, blockHeight, verifySignatures)
 
+	case TxnTypeCoinLockup:
+		// TODO: Once merged, update blockTimestamp parameter
+		totalInput, totalOutput, utxoOpsForTxn, err = bav._connectCoinLockup(txn, txHash, blockHeight, 0, verifySignatures)
```

Comment:
Normally when I do something like set an argument to zero I like to add a FIXME comment so that I make sure I don't miss it. Normally I treat "FIXME" as "DO NOT MERGE." Ideally we would have a hook that prevents merging as long as a FIXME is in your changes, but we're not that advanced right now...